packages/gatsby/src/utils/websocket-manager.ts

```diff
@@ -116,17 +121,47 @@ export class WebsocketManager {
       pingTimeout: 30000,
     })
 
+    const updateServerActivePaths = (): void => {
```

Comment:
Nit: Wouldn't this be a method on the class? Same goes for some other const arrows below...

Reply:
This should be private/internal - not sure if I can mark method as private and our tooling would honor that and don't let through usage of it outside of internals of it

Reply:
I checked both TS (`private`) and JS (`#`) private class fields and it seems like what we have right now doesn't catch that those are used outside of the class, so I would like to keep it as-is if this is not a huge concern